.github/CODEOWNERS

```diff
@@ -46,6 +46,9 @@
 # PRLabel: %Communication - Calling Server
 /sdk/communication/sdk/communication/azure-communication-callingserver/ @minwoolee-msft
 
+# PRLabel: %Communication - Calling Server
+/sdk/communication/sdk/communication/azure-communication-callingserver/ @juntuchen-msft @cochi2
```

Comment:
This is the same path as L47 above.   Please consolidate into one line.

Reply:
Thanks for pointing out, I have adjusted that now